test/verify/check-shell-multi-machine

```diff
@@ -706,25 +706,7 @@ class TestMultiMachine(MachineCase):
         b.logout()
         b.wait_visible("#login")
 
-        if m1.image in ["fedora-coreos"]:  # no semanage
-            m1.execute("setenforce 0")
-        else:
-            m1.execute("! selinuxenabled || semanage port -a -t ssh_port_t -p tcp 2222")
```

Comment:
Yep, this was surely a copy&paste error.

Reply:
Yeah, thanks for confirming.